Rubberduck.Refactorings/EncapsulateField/ConflictDetection/EncapsulateFieldConflictFinder.cs

```diff
@@ -268,14 +268,18 @@ private bool HasConflictWithLocalDeclarationIdentifiers(IEncapsulateFieldCandida
 
             //Only check IdentifierReferences in the declaring module because encapsulated field 
             //references in other modules will be module-qualified.
-            var candidateLocalReferences = candidate.Declaration.References.Where(rf => rf.QualifiedModuleName == candidate.QualifiedModuleName);
+            var candidateLocalReferences = candidate.Declaration.References.Where(rf => rf.QualifiedModuleName == candidate.QualifiedModuleName
+                && !UsesMeQualifier(rf));
 
             var localDeclarationConflictCandidates = membersToEvaluate.Where(localDec => candidateLocalReferences
                .Any(cr => cr.ParentScoping == localDec.ParentScopeDeclaration));
 
             return localDeclarationConflictCandidates.Any(m => m.IdentifierName.IsEquivalentVBAIdentifierTo(identifierToCompare));
         }
 
+        private static bool UsesMeQualifier(IdentifierReference identifierReference) 
+            => identifierReference.Context.Parent is VBAParser.MemberAccessExprContext || identifierReference.Context.Parent is VBAParser.WithMemberAccessExprContext;
```

Comment:
Something doesn't feel right about this condition. Where are we checking that the qualifier is, indeed, the reserved identifier `Me`?

Reply:
I think the condition here is actually correct, but the statement what the PR does is wrong, as well as the function's name. It should rather be called `IsQualified`.

We can actually have the same problem in two other ways. 
1. You can module qualify a field in a standard module. 
2. You can have another variable of the same class, e.g. as a parameter to a function, and access the field there.

I think these cases should have tests and the PR should change the statement about what it does.

Reply:
It does look a bit 'broad' when considered in isolation...Since this predicate function is only called from within `HasConflictWithLocalDeclarationIdentifiers` it's capture is narrowed by the caller (probably _not_ a best practice).  What I intend to do is relocate it as a local predicate function within `HasConflictWithLocalDeclarationIdentifiers` (to make it's capture scope clear) and change the name to `IsQualifiedReference`.  Finding the usage of the `Me` qualifier is not really the goal of the evaluation...the goal is to determine that the reference has a qualifier...and, any qualified reference is excused from further conflict evaluations.

The PR name:  This PR's scope limited to responding a bug report where a conflict is detected where none exists.  So, describing this a a False-positive seems appropriate.  I'm open to suggestions for a better name.

The other cases identified above are already addressed by the implementation of `EncapsulateFieldRefactoring` (EF).  As part of the refactoring, EF updates every reference in other modules to include a module qualification...So, external module references to the new Property do not require evaluation.  Internally, references to the new Property identifier (which is the 'old' field identifier references) are evaluated for conflicts with all members.  The bug here is that internal references that are 'qualified' ('Me'.*) are also exempt from providing a positive result for the same reason that external module qualified references are exempt.